beets/dbcore/db.py

```diff
@@ -1189,6 +1208,17 @@ def _make_table(self, table: str, fields: Mapping[str, types.Type]):
         with self.transaction() as tx:
             tx.script(setup_sql)
 
+        # Index handling
+        with self.transaction() as tx:
+            for index in indices:
+                if index.name in current_indices:
+                    continue
+
+                columns_str = ", ".join(index.columns)
+                tx.script(
+                    f"CREATE INDEX {index.name} ON {table} ({columns_str})"
+                )
```

Comment:
Based on my previous comment, I think it's also more clear to avoid GOTO equivalents like `continue`

```py

```suggestion
        current_indices = {row[1] for row in index_rows}
        with self.transaction() as tx:
            for index in indices:
                if index.name not in current_indices:
                    index.create(tx, table)

```

Reply:
I prefer to keep control flow simple and avoid deep nesting. That’s why I often lean toward early returns and continue statements, they help. 

Also I wanted the new code style to match the old code, which also used the continue. Maybe we want to change this around for all code in a more overarching PR?

Reply:
Note I considered suggesting to replace multiple `SELECT * FROM table/index_info` with a single `SELECT * FROM sqlite_master` query to return data about **_all_** models and indices immediately but thought this would probably be out of scope of our adjustment.